src/redemptions/redemption.service.ts

```diff
@@ -61,80 +76,106 @@ export class RedemptionService {
       };
     }
     if (
-      transactionStatus.workflow.status === 'SESSION_EXPIRED' ||
-      transactionStatus.workflow.status === 'TOKEN_EXPIRED' ||
-      transactionStatus.decision.type === DecisionStatus.NOT_EXECUTED
+      response.workflow.status === 'SESSION_EXPIRED' ||
+      response.workflow.status === 'TOKEN_EXPIRED' ||
+      response.decision.type === DecisionStatus.NOT_EXECUTED
     ) {
       return {
         status: KycStatus.TRY_AGAIN,
         failureMessage: null,
         idDetails,
       };
     }
-    if (transactionStatus.decision.type === DecisionStatus.PASSED) {
+    if (response.decision.type === DecisionStatus.PASSED) {
       return {
         status: KycStatus.SUBMITTED,
         failureMessage: null,
         idDetails,
       };
     }
+    const repeatedFaceWorkflowIds = this.getRepeatedFaceWorkflowIds(
+      response.capabilities.imageChecks,
+    );
+    const multiAccountFailure = await this.multiAccountFailure(
+      userId,
+      repeatedFaceWorkflowIds,
+    );
+
     // TODO: HANDLE WARN, use decision.risk.score?
-    const failure =
```

Comment:
Consolidated these into a single check since I had to pull out labels anyways for the call to `this.hasOnlyDuplicateFaceFailures(labels)`